rebar.config

```diff
@@ -21,17 +20,15 @@
 {deps_dir, "deps"}.
 {deps,
  [
-  {lager,      ".*",  {git, "https://github.com/basho/lager.git",          "3.0.2"}},
   {elvis_core, ".*",  {git, "https://github.com/inaka/elvis_core",         "0.2.7"}},
```

Comment:
You updated `elvis_core` in the `Makefile`, so please update it here. Also add `katana_code` here, please.